webgpu.h

```diff
@@ -1566,6 +1568,37 @@ typedef struct WGPUAdapterInfo {
     /*.deviceID=*/0 _wgpu_COMMA \
 })
 
+/**
+ * Enabled by the "subgroups" feature.
+ * Provides bounds on the sizes of subgroups supported by the adapter.
+ *
+ * Default values can be set using @ref WGPU_ADAPTER_PROPERTIES_SUBGROUPS_INIT as initializer.
+ */
+typedef struct WGPUAdapterPropertiesSubgroups {
+    WGPUChainedStructOut * nextInChain;
+    /**
+     * If the "subgroups" feature is supported, the minimum supported subgroup size for the adapter.
+     *
+     * Defaults to `0`.
```

Comment:
The spec PR makes the default values (even for unsupported) for min and max 4 and 128.